packages/docs/src/page-configs/getting-started/configuration-guide/code-examples/icons.ts

```diff
@@ -1,8 +1,7 @@
-export const iconsInstall = `
-yarn add material-design-icons-iconfont -D
-// {{ $t('all.code.or') }}
-npm install material-design-icons-iconfont -D
-`
+export const iconsInstallObject = {
```

Comment:
And update [configuration-guide#icons-fonts](https://vuestic.dev/en/getting-started/configuration-guide#icon-fonts)